### DISCUSSION TITLE
add a placeholder to list all members of an island

### DISCUSSION THREAD

**Discussion by Shottzyy** [BG-Software-LLC/SuperiorSkyblock2#231] [Pending Feature Requests] [answered]
There are placeholders for pretty much everything else but I could not find a placeholder to list all members of an island, for example %superior_island_members_list_1% would show all members of island top 1, this would be insanely helpful for creating your own custom menus, or use within discord bots (which is what I would like to use it for)

Why I want this added;
![image](https://user-images.githubusercontent.com/33270684/116003692-ee17b100-a5f6-11eb-8a61-2557fa23958d.png)
I would like to display the current members of an island, with their island worth and output it every few hours so I can easily monitor any rule breakers from boosting themselves very high, but this will also be very useful for menus and I think this should be added because it is likely already apart of the api so it should be easy to implement.

**Comment by Lemvs** [comment, marked as answer]
@OmerBenGera added - %superior_island_team_list% (This can also be used as %superior_island_top_<type>_<#>_<placeholder>%, so it's solved)
